source/people/main.rst

```diff
@@ -123,6 +115,7 @@ None at this time.
 
 Alumni
 -------------
+* Xiaokang Zhang, Visiting Student Researcher (2017-2019), University of Science and Technology of China
```

Comment:
This list is alphabetical by family name.  Please move to the correct position

Reply:
Done in https://github.com/CNERG/cnerg.github.com/pull/151/commits/e2ac79979b519d16d8a0b00c08568bef13c37a2b.